### ISSUE TITLE
Date range: calendar icon buttons missing text equivalent (1.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#841]
### Expected behavior

Calendar icon buttons have a text equivalent in a label - Select date with calendar
### Actual behavior

No button text label available.

**Comment by Blackbaud-PatrickOFriel** [comment]
The current datepicker control that we use from ui-bootstrap (the calendar that opens after you click the calendar icon button) does not behave very well with the keyboard right now (can't really navigate around properly with arrows/tabs). I see two options: 
1. upgrade our version of ui-bootstrap (which could cause all kinds of breaking changes that users could have to account for).
2. Put an aria-hidden on the calendar button because users can enter dates in the input without use of the datepicker. Then, when we port this component to SKY UX 2 we will create our own datepicker control that is accessible and does not rely on ui-bootstrap.

Thoughts?

**Comment by Blackbaud-MattGregg** [comment]
I hadn't gotten to keyboard testing this component because I realized automated testing hadn't been turned on yet. Waiting for SKY 2 or ui-bootstrap update sounds like a good approach.

The aria-hidden solution doesn't hide the button from people using their keyboard, just from assistive technologies like screen readers. 

I just tried the calendar with Voiceover and keyboard and it actually works for navigating and selecting dates. Semantics within the calendar aren't good, but one can actual move around the dates and months. Would third option be to add an aria-label to the button instead of aria-hidden (i.e., fix this issue and then just open one to track the problem(s) with the ui-bootstrap component itself)?

**Comment by Blackbaud-PatrickOFriel** [comment]
Option 3 sounds good to me

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.7.6](https://github.com/blackbaud/skyux/releases/tag/v1.7.6)
